### ISSUE TITLE
Provide greater control over how CSS is exported

### ISSUE THREAD

**Issue by RoelN** [Wakamai-Fondue/wakamai-fondue-engine#63]
Wakamai Fondue now exports _all_ the CSS is can. It would be great if you could toggle parts:

- @font-face 
- Unicode range
- Classes per named instance
- etc.

**Comment by dar5hak** [comment]
Yes, and it might also be useful to be able to select specific OpenType features. WF could then only include the classes and variables for these, resulting in a terser stylesheet.

I also hazard a guess that most people would not want to disable `kern` or `liga`. So those classes and variables might be going unused anyway.
